backend/src/lendings/lendings.controller.ts

```diff
@@ -23,6 +23,7 @@ export const create: RequestHandler = async (
       id,
       req.body.condition,
     );
+    res.status(201).json();
```

Comment:
```suggestion
    res.status(201).end();
```

[.json(), .send(), .end() 차이](https://velog.io/@yunsungyang-omc/nodejs-Express-res.send-res.json-res.end-%EB%B9%84%EA%B5%90) 가 있으니 우리 상태에선 아무것도 안 보내줘도 되니까 .end()로 보낼것을 추천합니다.

Reply:
오 좋습니다 변경해서 올리겠습니다!